test/hotspot/jtreg/runtime/appcds/cacheObject/ArchivedModuleWithCustomImageTest.java

```diff
@@ -25,15 +25,12 @@
  * @test
  * @summary Test archived module graph with custom runtime image
  * @requires vm.cds.archived.java.heap
- * @library /test/jdk/lib/testlibrary /test/lib /test/hotspot/jtreg/runtime/appcds
- * @modules java.base/jdk.internal.module
- *          java.management
- *          jdk.jlink
- *          jdk.compiler
```

Comment:
Includes the changes from [JDK-8229267](https://bugs.openjdk.org/browse/JDK-8229267) ( commit [e9b271d](https://github.com/openjdk/jdk/commit/e9b271dd1065994e4538e3a628e3f3542fdcacb5) ) on this file